apps/onboarding-functions/src/main/java/it/pagopa/selfcare/onboarding/service/NotificationServiceDefault.java

```diff
@@ -193,7 +203,13 @@ private void sendMailWithFile(List<String> destinationMail, String templateName,
             MailTemplate mailTemplate = objectMapper.readValue(template, MailTemplate.class);
             String html = StringSubstitutor.replace(mailTemplate.getBody(), mailParameters);
 
-            final String subject = String.format("%s: %s", prefixSubject, mailTemplate.getSubject());
+            final String subject;
+
+            if(Objects.nonNull(prefixSubject)) {
+                subject = String.format("%s: %s", prefixSubject, mailTemplate.getSubject());
+            } else {
+                subject = mailTemplate.getSubject();
+            }
```

Comment:
you could be more elegant using Optional.ofNullable...for ex. `subject = Optional.ofNullable(prefixSubject).map(value -> String.format("%s: %s", value, mailTemplate.getSubject())).orElse(mailTemplate.getSubject())`